Resources/Locale/en-US/hotpotato/hot-potato.ftl

```diff
@@ -0,0 +1,17 @@
+hot-potato-give = {CAPITALIZE(THE($user))} forces { THE($item) } onto {CAPITALIZE(THE($target))}!
```

Comment:
```suggestion
hot-potato-give = {CAPITALIZE(THE($user))} forces { THE($item) } onto {THE($target)}!
```

Reply:
capitalize capitalizes the first letter of whats passed in, this isnt correct usage